lessons/pydata/ml_intro/index.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>Není zavádějící, že "diabetes prediction" a "trend" jsou ta samá data jen jinak zobrazená?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
brala jsem, že trend je ta čára, predikce jsou v těch konkrétních bodech - t. j. tečky, ale si by to chtělo popsat líp